exercises/settings.gradle.kts

```diff
@@ -53,6 +53,7 @@ include(
     "practice:pangram",
     "practice:pascals-triangle",
     "practice:perfect-numbers",
+    "practice:protein-translation",
```

Comment:
This list looks lexicographical, so could you move this below `"practice:prime-factors"`, so the order is preserved?

Reply:
Well spotted! Thanks for reviewing! 😊